packages/pool/contracts/GetterUtils.sol

```diff
@@ -239,6 +240,48 @@ contract GetterUtils is StateUtils, IGetterUtils {
         return userDelegateAt(userAddress, block.number);
     }
 
+    /// @notice Called to get the current locked tokens of the user
+    /// @param userAddress User address
+    /// @return locked Current locked tokens of the user
+    function getUserLocked(address userAddress)
+        public
+        view
```

Comment:
Moved this here from `RewardUtils.sol`, also made it into a `view` method by removing the reward payment. Having to do static calls to view stuff is annoying.